README.md

```diff
@@ -1,9 +1,9 @@
-Emby.ApiClient
+Jellyfin.ApiClient
 ======================
 
-This portable class library makes it very easy to harness the power of the Embyr API. This is available as a Nuget package:
+This portable class library makes it very easy to harness the power of the Jellyfin API. ~~This is available as a Nuget package:~~
 
-[Emby.ApiClient](https://www.nuget.org/packages/MediaBrowser.ApiClient/)
+~~[Emby.ApiClient](https://www.nuget.org/packages/MediaBrowser.ApiClient/)~~ (Maybe sometime in the future)
```

Comment:
Should probably just remove the text instead of using strikethrough.

Reply:
Done in 3641a54